docker-storage-setup.conf

```diff
@@ -1,7 +1,28 @@
+# By default no new volume and fs will be setup for docker root dir. Docker
+# will create /var/lib/docker/ and use underlying fs for storing image and
+# containers.
+# 
+# If one wants to carve out a separate logical volume for storing docker
+# metadata and images/containers/volumes, then set DOCKER_ROOT_VOLUME=yes
+#
+# This will try to create a logical volume, create filesystem on that volume
+# and mount on $DOCKER_ROOT_DIR.
```

Comment:
I think very soon we will have request for allowing passing in filesystem to create and options to pass in. May be we can create two more options at that time. Say,

DOCKER_ROOT_FS="xfs"
DOCKER_ROOT_FS_OPTS="-o discard"

This is just an example for future extension. For now, I think we can just mention that by default xfs filesystem is created.